<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSWindow</class>
 <widget class="QMainWindow" name="MSWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManShop</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRefaire"/>
    <addaction name="separator"/>
    <addaction name="actionCouper"/>
    <addaction name="actionCopier"/>
    <addaction name="actionColler"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>263</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>524287</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="windowTitle">
    <string>Centre de Gestion</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolBox" name="tbControle">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="tabSpacing">
        <number>6</number>
       </property>
       <widget class="QWidget" name="produits">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>180</width>
          <height>382</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Produits</string>
        </attribute>
        <widget class="QToolButton" name="btNouvelleCategorie">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>131</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>Nouvelle Catégorie</string>
         </property>
        </widget>
        <widget class="QToolButton" name="btNouveauProduit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>240</y>
           <width>131</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>Nouveau Produit</string>
         </property>
        </widget>
        <widget class="QToolButton" name="btNouveauxProduits">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>310</y>
           <width>131</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>Nouveaux Produits</string>
         </property>
        </widget>
        <widget class="QToolButton" name="btRechercheCategorie">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>131</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>Recherche Catégorie</string>
         </property>
        </widget>
        <widget class="QToolButton" name="btSupprimerCategorie">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>131</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>Supprimer Catégorie</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="stock">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>180</width>
          <height>382</height>
         </rect>
        </property>
        <attribute name="icon">
         <iconset resource="ressources.qrc">
          <normaloff>:/images/stock.png</normaloff>:/images/stock.png</iconset>
        </attribute>
        <attribute name="label">
         <string>Stock</string>
        </attribute>
        <widget class="QToolButton" name="btNouveauStock">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>121</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Créer un nouveau stock</string>
         </property>
         <property name="statusTip">
          <string>Créer un nouveau stock</string>
         </property>
         <property name="text">
          <string>Nouveau Stock</string>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/images/nouveau_stock.png</normaloff>:/images/nouveau_stock.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QToolButton" name="btSupprimerStock">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>121</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Supprimer Stock</string>
         </property>
        </widget>
        <widget class="QToolButton" name="btAjouterProduit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>131</width>
           <height>81</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Nouveau Produit</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="commande">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>180</width>
          <height>382</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Commande</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="vente">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>180</width>
          <height>382</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Vente</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="A">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>180</width>
          <height>382</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Inventaire</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRefaire">
   <property name="text">
    <string>Refaire</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCouper">
   <property name="text">
    <string>Couper</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="text">
    <string>Copier</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="text">
    <string>Coller</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
